app/styles/app/modules/yaml.scss

```
@@ -41,11 +41,19 @@ pre.codedisplay[class*='language-'] {
   }
```

```suggestion
    color: $asphalt-grey;
  }
```
The `.codedisplay code` styles below  can be moved up in here

---

done, i think :)

---

Perfect! :smile: Looks good to me, feel free to merge at your convenience :rocket: